core/modules/block/block.admin.inc

```diff
@@ -13,15 +13,13 @@ function block_admin_list() {
 
   $header = array(
     t('Block'),
-    array('data' => t('Description'), 'class' => array('priority-low')),
```

Comment:
Why was this removed?

Reply:
@herbdool 
Thanks for reviewing!
I removed that column because by now there was only the static "A reusable custom block." 
But in connection to https://github.com/backdrop/backdrop-issues/issues/2722 it makes completely sense to print that column again. Thank you for pointing this out.

I removed the packaging infos from the info files an will update the PR.

Bjoern